recipes/_configure_indexes.rb

```diff
@@ -36,6 +36,20 @@
       :index
     end
 
+  daily_mb = hash.delete('_maxDailyDataSizeMB')
```

Comment:
Padding added to ingestion per day to account for the metadata. If the ingestion per day is 50MB and if we decided to add 10% padding, _maxDailyDataSizeMB should be 55MB.